src/data/finsupp/defs.lean

```diff
@@ -894,7 +894,12 @@ mul_hom_ext $ λ x, monoid_hom.congr_fun (H x)
 lemma map_range_add [add_zero_class N]
   {f : M → N} {hf : f 0 = 0} (hf' : ∀ x y, f (x + y) = f x + f y) (v₁ v₂ : α →₀ M) :
   map_range f hf (v₁ + v₂) = map_range f hf v₁ + map_range f hf v₂ :=
-ext $ λ a, by simp only [hf', add_apply, map_range_apply]
+ext $ λ _, by simp only [hf', add_apply, map_range_apply]
+
+lemma map_range_add' [add_zero_class N] [add_monoid_hom_class β M N]
+  {f : β} {hf : f 0 = 0} (hf' : ∀ x y, f (x + y) = f x + f y) (v₁ v₂ : α →₀ M) :
```

Comment:
The redundant assumptions `hf'` has not been removed (and elsewhere). You could even inline a proof of `f 0 = 0` but maybe that's pointless ...

Reply:
Lots on my mind today...